### ISSUE TITLE
Accessibility: Status images in the delegate table are not suitable for users with impaired acuity

### ISSUE THREAD

**Issue by alessiodf** [ArkEcosystem/explorer#695]
**Expected Behaviour**

Users should be **easily** able to tell the difference between the three statuses.

**Current Behaviour**

It is not possible for certain colourblind people with additional acuity problems to tell the difference. Try crossing your eyes and looking at the explorer, imagining there is no colour.

**Possible Solution**

1. Use different larger shapes. A full sized green square, red circle or yellow triangle for the respective statuses would provide more definition to the icons rather than dots and dashes. Different shapes would be more easily differentiated for users with acuity problems.

Alternatively:

2. Just use ✅, ❌ and ⚠️ unicode emoji which will render natively and already convey enough visual information to be understandable without trying to work out "what do two dots, a dot and a dash or two dashes mean" as they are already intuitive in context. Bonus points for the fact that they work natively with screen readers re: #693. No need to reinvent the wheel.

**Comment by ArkEcosystemBot** [comment]
**Thanks for opening this issue!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

**Comment by dated** [comment]
_stalebot, be gone!_

**Comment by ArkEcosystemBot** [comment]
This issue has been closed. If you wish to re-open it please provide additional information.
